gpkit/small_scripts.py

```diff
@@ -163,41 +163,55 @@ def sort_and_simplify(exps, cs):
     return tuple(matches.keys()), cs_
 
 
-def results_table(data, title, senss=False):
-    strs = ["              | " + title]
-    for var, val in sorted(data.items(), key=lambda x: str(x[0])):
-        if isinstance(val, Iterable):
-            vector = bool(val.shape)
-            if vector:
-                if all(val == val[0]):
-                    vector = False
-                    val = val[0]
-        else:
-            vector = False
+def results_table(data, title, minval=0, printunits=True, fixedcols=True,
+                  varfmt="%4s : ", valfmt="%-8.3g", vecfmt="%-7.2g"):
+    """
+    Pretty string representation of a dict of VarKeys
+    Iterable values are handled specially (partial printing)
+
+    Parameters
+    ----------
+    data: dict whose keys are VarKey's
+        data to represent in table
+    title: string
+    minval: float
+        skip values with all(abs(value)) < minval
+    printunits: bool
+    fixedcols: bool
+        print rhs (val, units, label) in fixed-width cols?
```

Comment:
Oh, because that's the question that the boolean parameter `fixedcols` answers. Ideas for better way to document welcome.